models/purchase.js

```diff
@@ -32,7 +32,10 @@ const purchaseSchema = mongoose.Schema(
         part_name: [String],
         subsystem: [String],
         price_per_unit: [Number],
-        quantity: [Number],
+        quantity: {
```

Comment:
Quantity should still be an array, also this provides a default at the database level not the UI level